src/apps/basic-transaction/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+// components
+import useBottomMenuModal from '../../hooks/useBottomMenuModal';
+import TextInput from '../../components/Form/TextInput';
+import Button from '../../components/Button';
+import FormGroup from '../../components/Form/FormGroup';
+import Label from '../../components/Form/Label';
+import Select from '../../components/Form/Select';
+
+// utils
+import { visibleChains } from '../../utils/blockchain';
+
+const App = () => {
+  const [t] = useTranslation();
+  const [destinationAddress, setDestinationAddress] = React.useState<string>('');
+  const [value, setValue] = React.useState<string>('');
+  const [data, setData] = React.useState<string>('');
+
+  const chainOptions = visibleChains.map((chain) => ({
+    id: `${chain.id}`,
+    title: chain.name,
+    value: chain.id,
+  }));
+
+  const [chainId, setChainId] = React.useState<number>(+chainOptions[0]?.value);
+  const { show } = useBottomMenuModal();
+
+  const send = () => {
+    if (!destinationAddress) return;
+    show({
+      type: 'send',
```

Comment:
Just a note for later, we should add an option to allow for:

- Immediate send: no possibility for users to queue their transactions from various apps, user must send now or never
- Batchable send: transaction is queued and user can choose to send it later providing the transaction isn't time sensitive (the app develop's responsibility"

Reply:
Immediate send can be done through exposed Prime SDK or TransactionKit components. I think the modal purpose is not for immediate send as modal shows something for user to confirm before it's sent.

Transaction queueing is another story that requires multiple other core features besides this, but yes, can be expanded.

Reply:
let's get a ticket written for sure